lib/allocator/index.spec.ts

```diff
@@ -1,41 +1,39 @@
-import Web3 from 'web3'
-import { CustomOptions } from '../option'
+import { ethers } from 'ethers'
 import { createAllocatorContract, CreateAllocatorContract } from '.'
 import { createCalculateMaxRewardsPerBlockCaller } from './calculateMaxRewardsPerBlock'
 import { allocatorAbi } from './abi'
 
+jest.mock('./calculateMaxRewardsPerBlock')
```

Comment:
@aggre 
I thought that calling the calculateMaxRewardsPerBlock function in this test would not be appropriate as a unit test, so I mocked it. What do you think?

Reply:
@abyssparanoia 
I would like to hear your opinion as well.

Reply:
I think this is good